framework/source/class/qx/test/Interface.js

```diff
@@ -365,7 +363,7 @@ qx.Class.define("qx.test.Interface",
             members :
             {
               getValue : function() {},
-              setValue : function(value) {}
```

Comment:
@pcdummy I think @derrell is right here. As the test is to check if a class fulfills the requirements of the interface it would be valid to have some code in the implementing method. Even such code like
```javascript
value += '';
```